### PR TITLE
Events

### PR DESCRIPTION
**Pull request by yayner2002**
in this PR, the following activities were done.

- Handle events in a React app.
- Use React life cycle methods.
- Understand the mechanism of lifting the state up.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,44 +1,98 @@
...
-            <button type="button">%</button>
-            <button type="button">÷</button>
+            <button type="button" id="AC" onClick={this.handleClick}>
+              AC
+            </button>
+            <button type="button" id="+/-" onClick={this.handleClick}>
+              +/-
+            </button>
+            <button type="button" id="%" onClick={this.handleClick}>
+              %
```
Comment (DeliceLydia) [inline src/components/Calculator.js:38]: - [x] This should be `modulus` not `division`

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @yayner2002, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Highlights for your project are:
- Awesome README
- Followed git workflow


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/components/Calculator.js (hunk 1 above)
Comment (yayner2002) [reply src/components/Calculator.js:38]: hi, @DeliceLydia I didn't get your comment. please make some clarification on this particular line of code. % is for modulus and it does the mod functionality as you said. besides, ÷ is for division and it does the division. if so, what are you asking me to do here? sorry if i did not get you easily  👍

Comment (DeliceLydia) [reply src/components/Calculator.js:38]: Hello @yayner2002, `%` should be modulus which means It should return the remainder of two integers that have been divided

Comment (yayner2002) [reply src/components/Calculator.js:38]: hi @DeliceLydia  yeah, it does what you are saying. I don't know if you have checked it. e.g.9%3 = 0 not 3 right ?

Comment (yayner2002) [reply src/components/Calculator.js:38]: hi, @DeliceLydia  are you still confused, I mean do I need to explain it further?

**Review by mikethreels — APPROVED** [review]
Hi Yayner, 

The modulus is working as expected on my side, Good job trying to clarify.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**